src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1765,15 +1765,15 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
         List<FeedbackQuestionAttributes> allQuestions =
                 fqLogic.getFeedbackQuestionsForSession(feedbackSessionName, courseId);
 
-        showAllQuestions(role, params, allQuestions);
+        Map<String, FeedbackQuestionAttributes> relevantQuestions = showAllQuestions(role, params, allQuestions);
```

Comment:
Some stuff that we can improve, for instance, this method checks if the user is an instructor